stories/notices.stories.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import SettingsNotice, { TYPE_WARNING, TYPE_INFO, TYPE_SUGGESTION } from '../assets/js/components/settings-notice';
+
+global._googlesitekitLegacyData.canAdsRun = true;
```

Comment:
Was this copied over from another stories file? I don't think it's relevant to notices.